Atlas lineage master

crazzysun opened the pull request:
---
Make sure to mark the boxes below before creating PR: [x]

- [ ] Description above provides context of the change
- [ ] Unit tests coverage for changes (not needed for documentation changes)
- [ ] Target Github ISSUE in description if exists
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions.
- [ ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

ashchegolev commented on airflow/config_templates/default_airflow.cfg:
```diff
@@ -309,14 +309,22 @@ auth_backend = airflow.api.auth.backend.default
 [atlas]
 sasl_enabled = False
-host =
+host = 192.168.31.46
```
Please remove your local IP address

crazzysun committed:
[AIRFLOW-8969] Atlas Backend module improvements

 - Create an Airflow operator atlas entity don't use execution date as part
   of the qualified name anymore.
 - Add config property for enabled/disabled creating missing inlets, outlets entities.
 - Add config property for enabled/disabled falling operator if sending lineage was failed
 - Add atlas timeout config property.
 - Add "template_fields" into airflow operator typedef and add additional config property
   for setting any additional operator attributes.
 - Fix DataSet class wrapper, add abstract types for file and JDBC source.
 - Added utils methods for correct generating inlets and outlet objects.
 - Added unit tests for creating missing entities and falling operator.

crazzysun committed:
Fixed import AirflowConfigException error, deleted airflow/utils/atlas
